modules/core/validation/models.js

```diff
@@ -88,6 +88,12 @@ export function validationIssue(attrs) {
 
 }
 
+validationIssue.ICONS = {
+    suggestion: 'info',
+    warning: 'alert',
+    error: 'error'
```

Comment:
Those are always used  like this…
Is there an advantage to have this split up and recomposed below?
It makes it easier to search those strings when they are pre-composed in one place.

```suggestion
    suggestion: '#iD-icon-info',
    warning: '#iD-icon-alert',
    error: '#iD-icon-error'
```

Reply:
yeah that seems helpful, done